edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -96,7 +97,7 @@ public static async Task<int> MainAsync(IConfiguration configuration)
 
             try
             {
-                mode = configuration.GetValue(Constants.ModeKey, "docker");
+                mode = configuration.GetValue(Constants.ModeKey, "iotedged");
```

Comment:
default mode should be "iotedged".

Reply:
Let's not change this right now. Default mode is docker by design.